code/datums/status_effects/xeno_buffs.dm

```diff
@@ -831,20 +831,17 @@
 
 /datum/status_effect/blessing/warding
 	id = "blessing of warding"
-	///A holder for the exact armor modified by this status effect
 	var/datum/armor/armor_modifier
 
 /datum/status_effect/blessing/warding/on_apply()
-	buff_owner = owner
-	if(!isxeno(buff_owner))
+	if(!isxeno(owner))
 		return FALSE
-	armor_modifier = buff_owner.soft_armor.scaleAllRatings(strength * 2.7)
-	buff_owner.soft_armor = buff_owner.soft_armor.attachArmor(armor_modifier)
+	armor_modifier = getArmor(14 * strength * 2.7 ,3 * strength * 2.7 ,5 * strength * 2.7 ,3 * strength *2.7 ,0 ,0 , 0 ,0 )//create armor for puppets
```

Comment:
why are you complicating it with this 2.7 mult, on top of the strength mult?

Reply:
cause this was the original calculation for the armor propably needs removing or tweaking but I dont want to mess with the balance only fix the bug

Reply:
this doesn't need to be a separate value, this does literally nothing of any use

Reply:
true I added it directly into the integer value